casper/src/test/scala/coop/rchain/casper/api/BondedStatusAPITest.scala

```diff
@@ -1,85 +1,184 @@
 package coop.rchain.casper.api
 
+import cats.effect.{Concurrent, Sync}
 import cats.syntax.all._
 import com.google.protobuf.ByteString
+import coop.rchain.blockstorage.BlockStore.BlockStore
+import coop.rchain.blockstorage.dag._
+import coop.rchain.casper.ValidatorIdentity
 import coop.rchain.casper.helper.BlockGenerator._
-import coop.rchain.casper.helper.TestNode.Effect
 import coop.rchain.casper.helper._
 import coop.rchain.casper.protocol.BlockMessage
-import coop.rchain.casper.util.ConstructDeploy
-import coop.rchain.casper.util.ConstructDeploy.basicDeployData
+import coop.rchain.casper.rholang.RuntimeManager
 import coop.rchain.casper.util.GenesisBuilder._
-import coop.rchain.crypto.PublicKey
 import coop.rchain.crypto.signatures.Secp256k1
-import coop.rchain.metrics.Metrics
+import coop.rchain.crypto.{PrivateKey, PublicKey}
+import coop.rchain.metrics.Span
+import coop.rchain.models.BlockHash.BlockHash
+import coop.rchain.models.FringeData
+import coop.rchain.models.Validator.Validator
+import coop.rchain.models.blockImplicits.getRandomBlock
+import coop.rchain.models.syntax._
+import coop.rchain.shared.Log
 import coop.rchain.shared.scalatestcontrib._
 import monix.eval.Task
-import monix.execution.Scheduler.Implicits.global
+import monix.testing.scalatest.MonixTaskTest
+import org.mockito.cats.IdiomaticMockitoCats
+import org.mockito.{ArgumentMatchersSugar, IdiomaticMockito}
 import org.scalatest.EitherValues
-import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.flatspec.AsyncFlatSpec
 import org.scalatest.matchers.should.Matchers
 
+import scala.collection.immutable.SortedMap
+
 class BondedStatusAPITest
-    extends AnyFlatSpec
+    extends AsyncFlatSpec
+    with MonixTaskTest
     with Matchers
     with EitherValues
     with BlockGenerator
     with BlockDagStorageFixture
-    with BlockApiFixture {
+    with BlockApiFixture
+    with IdiomaticMockito
+    with IdiomaticMockitoCats
+    with ArgumentMatchersSugar {
   // 4 nodes with 3 validators bonded
-  val keys              = randomValidatorKeyPairs.take(3).toList :+ ConstructDeploy.defaultKeyPair
-  val bonds             = createBonds(keys.map(_._2).take(3))
-  val genesisParameters = buildGenesisParameters(keys)(bonds)
-  val genesisContext    = buildGenesis(genesisParameters)
+  private val keys = randomValidatorKeyPairs.take(3).toList
+  private val initialComputeBondsResult = keys
+    .zip(createBonds(keys.map(_._2)))
+    .map { case ((_, pubKey), (_, bond)) => pubKey.bytes.toByteString -> bond }
+    .toMap
+  private val gB = {
+    getRandomBlock(
+      setBonds = initialComputeBondsResult.some,
+      setValidator = toValidatorOpt(keys.head._1)
+    )
+  }
```

Comment:
Braces are redundant, you save two lines.

Reply:
[Fixed](https://github.com/rchain/rchain/pull/3787/commits/7c4ee54fc3f7e886bf452a2e00b067f8f93edf3b).